packages/taquito-beacon-wallet/package.json

```diff
@@ -30,8 +30,9 @@
     "lint": "tslint  --project tsconfig.json -t codeFrame 'src/**/*.ts' 'test/**/*.ts'",
     "precommit": "lint-staged",
     "prebuild": "rimraf dist",
-    "build": "tsc --project ./tsconfig.prod.json --module commonjs && rollup -c rollup.config.ts ",
-    "start": "rollup -c rollup.config.ts -w"
+    "version-stamp": "node ../taquito/version-stamping.js",
+    "build": "npm run version-stamp && tsc --project ./tsconfig.prod.json --module commonjs && rollup -c rollup.config.ts ",
+    "start": "npm run version-stamp && rollup -c rollup.config.ts -w"
   },
```

Comment:
Is it a good idea to add a line: To all `package.json`s? This causes the file to be present right after `npm i`, probably avoiding any possible errors people might see before their first `npm start`

```
"postinstall": "npm run version-stamp"
```

Reply:
This is a great recommendation, I have added this.